services/navigation/client_test.go

```diff
@@ -204,6 +213,11 @@ func TestClient(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, paths, test.ShouldResemble, receviedPaths)
 
+		// test Properties
+		prop, err := workingNavClient.Properties(ctx)
+		test.That(t, err, test.ShouldBeNil)
+		test.That(t, prop.MapType, test.ShouldEqual, prop.MapType)
```

Comment:
typo?

Reply:
Ya, good catch, I used prop to fine both the output and what was used in the Func definition.